### PR TITLE
Updated deps

### PR DESCRIPTION
**Pull request by stewartbutler**
Updating deps after update to CNI (https://github.com/istio/cni/pull/257)

Done via `./bin/update_deps.sh`

### CODE REVIEW

**Comment by howardjohn** [comment]
/retest

I suspect it may be a regression by proxy update, attempting one more try

**Comment by stewartbutler** [comment]
OK. I was investigating the test to see if it was possible to update it, I retested it the first time to see if I could get more information out of it.

**Comment by howardjohn** [comment]
@gargnupur failures might be related to your changes? just a guess though

**Comment by gargnupur** [comment]
Yes, it's because of this PR: https://github.com/istio/proxy/commit/c32cb798758a6cd9c4e830c285833141ad69b20f

Basically in the query, we need to remove response_code and TCP metrics doesn't have that.. I fixed it in release-1.5 -> https://github.com/istio/istio/blob/66fbccea98568fdc0a9e00380abb80fb869adf72/tests/integration/telemetry/stats/prometheus/tcp/stats_tcp_filter_test.go

File: istio.deps [hunk 1]
```diff
@@ -4,13 +4,13 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "3a04f855cd176213f316b5f12d67c2a79e2ffdb9"
+    "lastStableSHA": "e75aa441f21df5f13e26f8517848616f10ebcec4"
```
Comment (yangminzhu) [inline istio.deps]: Can we use the latest sha (`5789f684ac335043e42b4a0242b42d4f48c3a0cd`)?

**Comment by howardjohn** [comment]
/retest

fixed in another pr

File: istio.deps (hunk 1 above)
Comment (stewartbutler) [reply istio.deps]: Done.

**Pull request merged**